docs/upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -192,14 +192,14 @@ There you can find links to upgrade notes for other versions too.
 
       <import file="${path.framework}/build.xml"/>
     ```
-- increase your PHPStan level to 4 in your `build.xml` ([#1040](https://github.com/shopsys/shopsys/pull/1040))
+- increase your PHPStan level to 4 in your `build.xml` and address all the reported violations ([#1381](https://github.com/shopsys/shopsys/pull/1381))
```

Comment:
I would recommend rephrasing this instruction since there is no obligation or urgency
```suggestion
- for the better quality of code in your project we recommend you to increase your PHPStan level to 4 in your `build.xml` and address all the reported violations ([#1381](https://github.com/shopsys/shopsys/pull/1381))
```